content/doc/addons/metabase.md

```diff
@@ -0,0 +1,181 @@
+
+## Create a Metabase add-on
+
+### From the Console
+
+1. [Create a new add-on](https://console.clever-cloud.com/users/me/addons/new) by clicking the **Create...** dropdown in the sidebar and then **an add-on**
+2. Select the Metabase add-on
+3. You can skip linking the add-on to an application, it won't be needed (connecting data sources to Metabase is done via Metabase's UI)
+4. Enter a name for your Metabase add-on and select the zone where you want it to be deployed
+
+### Using the CLI
+
+1. Make sure you have `clever-tools` installed locally. Please refer to the [setup guide](https://github.com/CleverCloud/clever-tools/blob/master/docs/setup-systems.md) if needed
+2. List the available plans and options for Metabase: `clever addon providers show metabase`
+3. In your terminal, run `clever addon create metabase <name> --org <org>` (`--org` is optional)
+
+Refer to the [Clever Tools documentation](https://github.com/CleverCloud/clever-tools/blob/master/docs/addons-backups.md) for more details on add-on management.
+
+## Acessing the Metabase Interface
+
+Once deployed, Metabase is accessible through an URL provided in the add-on's `Service dependencies` page. You can also go in the environment variables page of the Java application and look for `MB_SITE_URL` value. To get this URL with Clever Tools, use the add-on ID:
+
+```
+$ clever addon env ADDON_ID
+METABASE_URL="https://an_id-metabase.services.clever-cloud.com"
+````
+
+## Underlying Resources
+
+When you create the Metabase add-on, Clever Cloud automatically deploys:
+
+- A [Java](/doc/applications/java/java-jar/) instance with Metabase pre-loaded
+- A [PostgreSQL](/doc/addons/postgresql/) database (for internal Metabase use)
+
+When you first connect to your Metabase instance, you will be taken to a wizard to create the first user, data sources, etc. Then you can configure and use it.
+
+* [Learn how to use Metabase](https://www.metabase.com/learn/metabase-basics/getting-started/find-data)
+
+## Plan sizing
+
+By default, Metabase on Clever Cloud uses small-size resources, i.e:
+
+- XS Java
+- XXS Small Space PostgreSQL
+
+They are dimensioned to suit a majority of needs. You can however manage and adjust them directly [in the Console](https://console.clever-cloud.com/) to fit your needs. You can for example change their settings, migrate to a larger storage database, etc. Vertical auto-scalability is available for this service. Different plans for Java and PostgreSQL are available on Clever Cloud.
+
+Metabase default resources are dimensioned to suit low concurrent usage of Metabase. When you have multiple users loading large dashboards concurrently, if your instance experiences crashes due to Out of Memory (OOM) issues, you should use a larger flavor for the Java application.
```

Comment:
Maybe this paragraph should replace/be merged with the previous one? It feels like a partial repetition... (cc @davlgd)